Components/weather.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import {Text, View, ScrollView} from 'react-native';
+
+import {getWeatherFromApi} from '../API/MDBApi';
+import {CityWeather} from './weather.type';
+
+interface State {
+  weather: CityWeather;
+}
+
+interface Props {}
+
+export class Weather extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      weather: {cod: '', message: 0, cnt: 0, list: null, city: null},
+    };
+  }
+
+  componentDidMount = () => {
+    this._loadWeather();
+  };
+
+  _loadWeather = () => {
```

Comment:
Why is there an underscore here ?

Reply:
This is to inform that the method is only used in this method, as if it were private in java. I don't know if it's very useful in js